pkg/config/config.go

```diff
@@ -64,6 +64,10 @@ func GetSystemSettingsAllowList() []string {
 		"overcommit-config",
 		"vip-pools",
 		"auto-disk-provision-paths",
+		"csi-driver-config",
+		"containerd-registry",
+		"storage-network",
+		"harvester-csi-ccm-versions",
```

Comment:
don't include the `csi-driver-config` and `harvester-csi-ccm-versions` as they are preserved as system configuration.

Reply:
Ok. I synced them because all the other options were there as well. But I understand that we don't need to add all of them. I will remove them.